app/scenes/DocumentDelete.tsx

```diff
@@ -44,12 +44,16 @@ function DocumentDelete({ document, onSubmit }: Props) {
 
             if (parent) {
               history.push(documentUrl(parent));
-              return;
```

Comment:
You could achieve the same thing by called `onSubmit();` here before the return statement with no other changes

Reply:
Brilliant!  Giving it a try now.

Reply:
Tested with your suggestion and it works great.  Thanks!